packages/nvidia_nat_test/src/nat/test/llm.py

```
@@ -66,14 +66,14 @@ async def async_sleep(self) -> None:
         await asyncio.sleep(self._delay_ms / 1000.0)
 
 
-@register_llm_provider(config_type=TestLLMConfig)
-async def test_llm_provider(config: TestLLMConfig, builder: Builder):
+@register_llm_provider(config_type=MockLLMConfig)
+async def test_llm_provider(config: MockLLMConfig, builder: Builder) -> AsyncGenerator[LLMProviderInfo, None]:
```

Ruff (0.14.10)

Unused function argument: `builder`

(ARG001)